package.json

```diff
@@ -18,13 +18,13 @@
     "clean": "rimraf dist/",
     "lint": "balena-lint --typescript src/ test/ typings/ init.ts index.js && tsc --noEmit --project .",
     "prettify": "balena-lint --typescript --fix src/ test/ typings/ init.ts index.js",
-    "prepare": "npm run build",
+    "prepack": "npm run build",
```

Comment:
This change means `npm install` within the repo won't trigger a build but it will still build before publishing and build on `npm install $giturl`